setup/helpers/perf_helper.rb

```diff
@@ -393,19 +420,20 @@ def classify_foss_nodes(host)
       # need to restart the server to make this change take effect.
       # TODO: should move the restart into a helper method or something, this
       # is duplicated in 99_restart_server.rb.
-      service_name = get_puppet_server_service_name_from_env()
+      service_name = get_puppet_server_service_name_from_env
 
       on(host, "systemctl restart #{service_name}")
 
       Beaker::Log.notify("Finished restarting service #{service_name}")
 
       # now update the value for environments dir
-      environments = on(host, puppet('config print environmentpath')).stdout.chomp
+      environments = get_puppet_config(host, "environmentpath")
```

Comment:
It looks like this was renamed to `puppet_config`.

Reply:
Thanks :100: for spotting this @billclaytor . I have pushed an updated version of the file with this corrected.